packages/fxa-auth-server/lib/payments/paypal-client.ts

```diff
@@ -132,7 +138,7 @@ export type NVPDoReferenceTransactionResponse = NVPResponse &
 
 export type NVPBAUpdateTransactionResponse = NVPResponse & BAUpdateData;
 
-export type NVPTransactionSearchResponse = NVPResponse & TransactionSearchData;
+export type NVPTransactionSearchResponse = TransactionSearchData & NVPResponse;
```

Comment:
These both now have a `L` attribute, so to ensure the caller gets the correct TS one out first we have to switch the order.